tslint.json

```diff
@@ -1,3 +1,34 @@
 {
-  "extends": "@blackbaud/skyux-builder/tslint"
+  "extends": "@blackbaud/skyux-builder/tslint",
+  "rules": {
```

Comment:
What's the logic for adding these rules here vx adding them to the baseline tslint file in builder?

Reply:
@Blackbaud-BobbyEarl Wouldn't adding them to builder create a "breaking change"? I definitely want to add these to Builder 2.x...

Reply:
If you'd prefer that we remove these rules until we've released Builder 2, that's fine too!